extensions/TheShovel/ShovelUtilsTW.js

```diff
@@ -0,0 +1,257 @@
+console.log("ShovelUtils v1.1")
+tempImageLoad = null
+string = null;
+temp = null;
```

Comment:
```suggestion
var temp = {};
```
fixes temp is not defined